base_tier_validation/tests/test_tier_validation.py

```diff
@@ -218,7 +218,7 @@ def test_12_approve_sequence(self):
                 "reviewer_id": self.test_user_1.id,
                 "definition_domain": "[('test_field', '>', 1.0)]",
                 "approve_sequence": True,
-                "sequence": 30,
```

Comment:
In current implementation, higher sequence here will get lower sequence here.
Meaning, when you set sequence 30, 20, 10 here. The review table will be 1, 2, 3.

I think this is by design AFSIK.

Reply:
In this pr，I have change to : higher sequence here will get higher sequence.

Reply:
This issue has been discussed over time, I don't remember the detail, may be we should check with the original author first. :sweat_smile: 
cc @LoisRForgeFlow

Reply:
I agree with you. 🙂
I have modify this module to use in my system, and it runs well.😊

Reply:
Great, so you can make some extra module on your own. :+1: 

Can you close this PR then. Thanks!

Reply:
OK